### ISSUE TITLE
Add Hover Effects to Contact Us Form

### ISSUE THREAD

**Issue by neeru24** [swaraj-das/Collect-your-GamingTools#462]
### Description

The "Contact Us" form currently lacks hover effects, making it feel static. Adding hover effects to buttons and input fields will enhance the form's interactivity and user experience.

Expected Behavior:

- Apply hover effects to the submit button and input fields to make the form more engaging.
- Hover effects could include color changes or subtle shadow effects for visual feedback.

### Screenshots

![image](https://github.com/user-attachments/assets/1a6337ef-11b4-46e5-9a06-4fb3e354ff1f)

### Checklist

- [X] I have checked the existing [issues](https://github.com/swaraj-das/Collect-your-GamingTools/issues).
- [X] I have read the [Code Of Conduct](https://github.com/swaraj-das/Collect-your-GamingTools/blob/main/CODE_OF_CONDUCT.md)
- [X] The changes don't break the code
- [X] Please attach the screenshots and video of the development work once it is done

**Comment by biswa2005** [comment]
Assign me this issue

**Comment by biswa2005** [comment]
Can you give me your WhatsApp number so that we can discuss about that more
clearly

**Comment by biswa2005** [comment]
please close this issue

**Comment by biswa2005** [comment]
this is already done.
